edgelet/edgelet-hsm/tests/crypto_create_cert_success.rs

```diff
@@ -3,15 +3,24 @@
 #![deny(unused_extern_crates, warnings)]
 #![deny(clippy::all, clippy::pedantic)]
 
+use std::env;
+use tempfile::TempDir;
+
 use edgelet_core::{
     Certificate, CertificateIssuer, CertificateProperties, CertificateType, CreateCertificate,
     KeyBytes, PrivateKey, Signature, IOTEDGED_CA_ALIAS,
 };
 use edgelet_hsm::Crypto;
 
+const HOMEDIR_KEY: &str = "IOTEDGE_HOMEDIR";
```

Comment:
It would be nice if this were set as a public const in hsm, and everywhere we need it referenced it from the HSM lib.